cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -193,6 +195,27 @@ public static Kafka createKafka(String namespace, String name, int replicas,
                 .build();
     }
 
+    public static Secret createClusterSecret(String clusterNamespace, String clusterName, int replicas, String secretName,
+                                             String caCert, String caKey, String caStore, String caStorePassword) {
+        Map<String, String> data = new HashMap<>(replicas * 4);
```

Comment:
can you please elaborate more? not sure what you mean by the load factor here

Reply:
By default `java.util.HashMap` has a load factor of 0.75. The Javadocs describe the relationship between capacity and load factor like this:

>  When the number of entries in the hash table exceeds the product of the load factor and the current capacity, the hash table is rehashed (that is, internal data structures are rebuilt) so that the hash table has approximately twice the number of buckets. 

So if you're expecting `data` to end up with `replicas * 4` keys in it and you asked for an initial capacity of `replicas * 4` then while populating `data` you'll end up rehashing once you've added `0.75 * replicas * 4` keys. To avoid rehashing the initial capacity would need to be `ceil(replicas * 4 / 0.75)`, or `(int) (1 + replicas * 4 /0.75)` (i.e. I got it wrong too in my initial comment).

This is something that we get wrong lots elsewhere in Strimzi. If we were serious about this we'd add a utility function to encapsulate the logic.

Reply:
Thanks! I found another constructor which accepts a load factor. That sounds more readable in this case IMHO.

`new HashMap<>(replicas * 4, 1)`

wdyt?